metals/src/main/scala/scala/meta/internal/metals/ProcessOutput.scala

```diff
@@ -0,0 +1,74 @@
+package scala.meta.internal.metals
+
+import java.lang.StringBuilder
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+import fansi.ErrorMode
+
+/**
+ * Handles streaming console output from a system process with potential ANSI codes.
+ *
+ * @param onLine The callback handler when the process has printed a single line.
+ *               Guaranteed to have no newline \n characters.
+ */
+class ProcessOutput(onLine: String => Unit) {
+  private var buffer = new StringBuilder()
```

Comment:
val and then `buffer.clear()`?

Reply:
I tried finding a `clear()` method on `java.lang.StringBuilder` but couldn't find any.

Reply:
It's in scala.collection.mutable.StringBuilder